### PR TITLE
Add 1d expiration to tmp operator-sdk images

### PR DESCRIPTION
**Pull request by tkrishtop**
build-depends: https://github.com/operator-framework/operator-sdk/pull/6529
build-depends: https://softwarefactory-project.io/r/c/dci-pipeline/+/29700

### CODE REVIEW

**Comment by tkrishtop** [comment]
check workload operator-sdk

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla operator-sdk

**Comment by tkrishtop** [comment]
check workload operator-sdk

File: roles/operator_sdk/tasks/build_scorecard_image.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     cmd: >
       podman buildx build
       -t {{ operator_sdk_img }}
+      -l quay.expires-after=2w
```
[the remark this answers is not in the document]
Comment (betoredhat) [reply roles/operator_sdk/tasks/build_scorecard_image.yml:33]: Nevermind, I see it is there with --label

```
podman buildx build --help|grep label
      --identity-label                               add default identity label (default true)
      --label stringArray                            set metadata for an image (default [])

```

[the remark this answers is not in the document]
Comment (tkrishtop) [reply roles/operator_sdk/tasks/build_scorecard_image.yml:33]: Hi Beto, thanks for checking, and sorry for the confusing test, I was just exploring if webhook for this repo works :) 
To have the image built, I need to use an operator-sdk PR in the dependencies, I'll add the dependencies and run this test asap.

**Comment by tkrishtop** [comment]
check workload operator-sdk

**Comment by tkrishtop** [comment]
> LGTM, do you know what is the default expiration time? I was trying to find, some docs show 14 days, but I was not sure if it depends on the version of quay or its configuration.

By default, the expiration date in our local quay is [never](https://172.27.17.53:4443/repository/operator-framework/operator-sdk?tab=tags).

**Comment by tkrishtop** [comment]
> * SUCCESS https://www.distributed-ci.io/jobs/00c281d7-c3c8-43b2-b098-7d62c14ad0ce/jobStates
>   from [Add 2w expiration to tmp operator-sdk images #15](https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/15)

ok, that did not work, because I messed up with dependencies.

**Comment by tkrishtop** [comment]
check workload operator-sdk

**Comment by tkrishtop** [comment]
The expiration label was added 
image

The overall job failed because podman_image module does not sync fast enough, and think that image is still used by a pod. The problem is that pod is already deleted and its namespace as well. Going to replace podman_image with `podman rmi -f` to fix the issue.

**Comment by tkrishtop** [comment]
check workload operator-sdk

**Review by betoredhat — APPROVED** [review]
Thank, I see the expiration dates in Quay.

**Comment by tkrishtop** [comment]
recheck

**Pull request merged**